ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -336,46 +397,93 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
+    quality,
+    filters,
+  }
+}
+
+const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById }) => {
+  if (!variant.genotypes) {
+    return null
+  }
+
+  let genotypes = variant.genotypes[individual.individualGuid]
+  if (!genotypes) {
+    return null
+  }
+  genotypes = Array.isArray(genotypes) ? genotypes : [genotypes]
+
+  const sampleTypeOrder = [SAMPLE_TYPE_GENOME, SAMPLE_TYPE_EXOME]
+  genotypes = genotypes.sort(
+    (a, b) => sampleTypeOrder.indexOf(a.sampleType) - sampleTypeOrder.indexOf(b.sampleType),
+  ).map(
+    genotype => getAllGenotypeDetails(genotype, variant, individual, isCompoundHet, genesById),
+  )
+  const allDetails = genotypes.map(info => info.details).reduce((acc, details, index) => {
+    if (index > 0) {
+      acc.push(<Divider />)
+    }
+    return acc.concat(details)
+  }, [])
+
+  const firstGenotype = genotypes[0]
+  const legacyOtherSampleGenotypes = useMemo(() => firstGenotype.genotype?.otherSample && [
+    { genotype: firstGenotype.genotype.otherSample, isHemiX: firstGenotype.isHemiX },
+  ], [firstGenotype])
+
   const content = (
     <span>
-      {genotype.otherSample && (
+      {firstGenotype.genotype?.otherSample && (
         <Popup
           header="Additional Sample Type"
-          trigger={<Icon name="plus circle" color={hasConflictingNumAlt ? 'red' : 'green'} />}
+          trigger={<Icon name="plus circle" color={firstGenotype.hasConflictingNumAlt ? 'red' : 'green'} />}
           content={
             <div>
-              {hasConflictingNumAlt && (
+              {firstGenotype.hasConflictingNumAlt && (
                 <div>
                   <VerticalSpacer height={5} />
-                  <Alleles genotype={genotype.otherSample} variant={variant} isHemiX={isHemiX} />
+                  <Alleles variant={variant} genotypes={legacyOtherSampleGenotypes} />
                   <VerticalSpacer height={5} />
                 </div>
               )}
-              {genotypeDetails(genotype.otherSample, variant, genesById)}
+              {genotypeDetails(firstGenotype.genotype.otherSample, variant, genesById)}
             </div>
           }
         />
       )}
-      <Alleles genotype={genotype} variant={variant} isHemiX={isHemiX} warning={warning} />
-      {previousCall && (
-        <Popup
-          content={previousCall.hover}
-          position="bottom"
-          trigger={<Label horizontal size="mini" content={previousCall.content} color={previousCall.color} />}
-        />
-      )}
-      {Number.isInteger(quality) ? quality : '-'}
-      {showSecondaryQuality && `, ${secondaryQuality ? secondaryQuality.toPrecision(2) : '-'}`}
-      {filters && (
-        <small>
-          <br />
-          {filters}
-        </small>
-      )}
+      <Alleles variant={variant} genotypes={genotypes} />
+      {genotypes.map(({ genotype, quality, showSecondaryQuality, secondaryQuality, previousCall, filters }) => (
+        <div key={genotype.sampleType}>
```

Comment:
for elaticsearch and old saved variants theres not necessarily a `sampleType` for the genotype - use an `|| sampleId` or something to ensure the key is always defined